src/api/notifications/forgetPushToken.js

```diff
@@ -7,7 +7,7 @@ import { apiDelete } from '../apiFetch';
  *
  * @param mobileOS - Choose the server-side API intended for iOS or Android clients.
  */
-export default (auth: Auth, mobileOS: 'ios' | 'android', token: string) => {
+export default (auth: Auth, mobileOS: 'ios' | 'android', token: string): Promise<empty> => {
```

Comment:
And for this one and `savePushToken`, it looks like we don't actually inspect the returned result at all -- so this could just be `Promise<mixed>`.